content/docs/user-guide/building/conda.md

```diff
@@ -0,0 +1,99 @@
+# Conda Environments
+
+Create a conda environment using requirements gathered from a model.
```

Comment:
When you're going to provide a user case, give a link to Conda and describe in 1-2 sentences what it is)

Reply:
Hm I guess I have the same feedback for all 3 new pages: put in the index page? Different section? Unclear why this is considered exporting.

Reply:
> Unclear why this is considered exporting.

Good point! Considering it, these pages look more suitable to be nested here to [user-guide/model](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/models). WDYT?

Reply:
Or we can indeed keep them here and clarify why they're located here.

Reply:
> these pages look more suitable to be nested here to user-guide/model

➕

Reply:
Another option (as I suggested) is to rewrite [this section's index page](https://mlem-ai-requirement-bui-vdivzc.herokuapp.com/doc/user-guide/building) and list these 3 as sections instead of sub-pages. Each one would have to be considerably shorter though, but I think that's doable since they have some repetition anyway. Code blocks can be shortened, etc. Up to you

Reply:
Yep, they're builders, but I don't think users would search for them in this section TBH. `Working with models` is a much likely place when I can expect to see this. If you would like to give a hint about a connection since this is a `builder` functionality, I think you can add links to both directions between pages themselves and the tutorial for `build` (`/doc/user-guide/building`).

Reply:
Where do you suggest to put this? Because there are pages pertaining to `tensorflow` / `torch` etc. And I don't think it fits in with those. IMO, it is much better to just change the title from `Exporting` to `Builders` I guess. And maybe change the contents of `index.md` for that section I guess.

Because `conda`, `venv` etc. are still related to other builders such as `docker` for example.
Putting it with `sklearn` etc. seems weird to me.

Reply:
Also, none of the files `sklearn`, `torch`, etc. are included in `sidebar.json` -- seems like all of them are incomplete and have `TODOs` in them. Is this intentional to not make all those files a part of the release?

Reply:
Good question. Yes, this is intentional. I don't think we will introduce them for quite a while - we have way to much stuff to do. Ok, let's do this:
1. Rename "Exporting models" to "Building models" - in sidebar and in header
2. Put a brief explanation in "Building models" why reqs/conda/venv is considered building
3. Mention venv/conda on "Working with models" page, so it could be more easily found

WDYT?

Reply:
done